### ISSUE TITLE
Error deserialize closed issue json

### ISSUE THREAD

**Issue by MurashovAlexander** [nmklotas/GitLabApiClient#50]
[Issue.cs#L40](https://github.com/nmklotas/GitLabApiClient/blob/master/src/GitLabApiClient/Models/Issues/Responses/Issue.cs#L40):
```csharp
...
[JsonProperty("closed_by")]
public string ClosedBy { get; set; }
...
```

[GitLab API Documentation](https://gitlab.com/help/api/issues.md#list-project-issues):  
```json
...
   "closed_by" : {
      "state" : "active",
      "web_url" : "https://gitlab.example.com/root",
      "avatar_url" : null,
      "username" : "root",
      "id" : 1,
      "name" : "Administrator"
    },
...
```

Please, fix it

```csharp
[JsonProperty("closed_by")]
public ClosedBy ClosedBy { get; set; }
```

```csharp
public class ClosedBy
{
    [JsonProperty("state")]
    public string State { get; set; }

    [JsonProperty("web_url")]
    public Uri WebUrl { get; set; }

    [JsonProperty("avatar_url")]
    public Uri AvatarUrl { get; set; }

    [JsonProperty("username")]
    public string Username { get; set; }

    [JsonProperty("id")]
    public int Id { get; set; }

    [JsonProperty("name")]
    public string Name { get; set; }
}
```

**Comment by dnagl** [comment]
@MurashovAlexander could you please give some more information about the exception?

**Comment by MurashovAlexander** [comment]
> @MurashovAlexander could you please give some more information about the exception?

JsonReaderException: Unexpected Character encountered while parsing value: {.Path '[0].closed_by', line 1, position 538.

**Comment by dnagl** [comment]
I'll have a look at it.

**Comment by dnagl** [comment]
Issue should be fixed in PR #53

**Comment by MurashovAlexander** [comment]
> Issue should be fixed in PR #53

thanks
